go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/golang/mock v1.4.0
 	github.com/golang/snappy v0.0.1 // indirect
 	github.com/google/go-github/v28 v28.1.1
+	github.com/goraz/onion v0.1.2
```

Comment:
Can you add a replace to your onion fork? (see the replaces below)

Reply:
Oh! I added a replace with my local fork, but ok, I'll add it for everyone!